src/schema-parser/asyncapi-schema-parser.ts

```diff
@@ -81,7 +82,7 @@ function getSchemaValidator(version: string): ValidateFunction {
  * To validate the schema of the payload we just need a small portion of official AsyncAPI spec JSON Schema, the Schema Object in particular. The definition of Schema Object must be
  * included in the returned JSON Schema.
  */
-function preparePayloadSchema(asyncapiSchema: AsyncAPISchema, version: string): AsyncAPISchema {
+function preparePayloadSchema(asyncapiSchema: JSONSchema7, version: string): JSONSchema7 {
```

Comment:
But AsyncAPISchema !== JSONSchema7, isnt? We extend from it, adding few extra fields like discriminator, etc.

Reply:
Right, I need to change it to the `AsyncAPISchemaDefinition` type. Thanks!

Reply:
Done.